### PR TITLE
Add possibility to run integration tests in parallel

### PR DESCRIPTION
**Pull request by jsafrane**
- add env. variable with etcd URL to intergration tests
- update documentation with example how to use it to find flakes

### CODE REVIEW

File: docs/devel/flaky-tests.md [hunk 1]
```diff
@@ -194,6 +194,41 @@ kubectl delete replicationcontroller flakecontroller
 
 If you do a final check for flakes with `docker ps -a`, ignore tasks that exited -1, since that's what happens when you stop the replication controller.
 
+
+# Reproducing integration test flakes
```
Comment (saad-ali) [inline docs/devel/flaky-tests.md]: Title seems misleading. Contents talks about starting multiple etcd servers. It doesn't explain how this ties into the title.

Comment (jsafrane) [inline docs/devel/flaky-tests.md]: It's not only about etcd but also about starting multiple integration test processes. I updated the text a bit.

**Comment by saad-ali** [comment]
LGTM

**Comment by jsafrane** [comment]
rebased

**Comment by gmarek** [comment]
This change broke kubemarks. Fix in #27719

**Pull request merged**